bin/detect

```diff
@@ -9,7 +9,7 @@ cat << EOF > "$2"
 name = "riff-node"
 
 [requires.metadata]
-artifact = "/layers/salesforce_nodejs-fn/middleware/dist"
+artifact = "../../../layers/salesforce_nodejs-fn/middleware/dist"
```

Comment:
How does this work with both `/workspace` and `/workspace/my-func`?

Reply:
For example:

```
▹ pwd
/tmp/myfunc
▹ cd ../../../dev
▹ pwd
/dev

▹ pwd
/tmp
▹ cd ../../../dev
▹ pwd
/dev
```

Reply:
ohh, I see. 🤦  you are going all the way up to the root, so the last `../` doesn't matter. I bet we could figure out the depth of the `pwd` and generate the right amount but this is straight forward enough.